dace/sdfg/utils.py

```diff
@@ -641,9 +644,10 @@ def remove_edge_and_dangling_path(state: SDFGState, edge: MultiConnectorEdge):
         else:
             if e.dst_conn:
                 e.dst.remove_in_connector(e.dst_conn)
-                e.src.remove_out_connector('OUT' + e.dst_conn[2:])
+                e.dst.remove_out_connector('OUT' + e.dst_conn[2:])
 
         # Continue traversing upwards
+        # TODO: What is about the children of the tree.
```

Comment:
What does this mean?

Reply:
It was a question for you (not you personally but the DaCe Developer).
I was just wondering, why the children where not handled?

Reply:
Okay this makes sense.

Reply:
Updated.